### PR TITLE
Taps for SSB Klass and PxWebApi

### PR DESCRIPTION
**Pull request by hholgersen**
Statistics Norway maintains a system named "Klass" for access to common classifications and code lists (municipalities, occupational codes, sector codes etc). The logo is the logo of Statistics Norway.

Several Nordic countries including Norway, Sweden and Finland use the open PxWebApi framework for publishing official statistics. The API is the same for all these countries. The PxWebApi doesn't have it's own logo, so I used the Logo of Statistics Sweden, who maintains the software, instead.

### CODE REVIEW

File: _data/meltano/extractors/tap-pxwebapi/storebrand.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+capabilities: []
+description: PxWebApi - API for official statistics used by Statistical offices in
+  several Nordic countries
+domain_url: https://www.ssb.no/en/api/pxwebapi
+executable: tap-pxwebapi
+keywords: []
```
Comment (pnadolny13) [inline _data/meltano/extractors/tap-pxwebapi/storebrand.yml:6]:
```suggestion
keywords:
- meltano_sdk
```

File: _data/meltano/extractors/tap-ssb-klass/storebrand.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
+capabilities: []
+description: Statistics Norway Klass API
+domain_url: https://data.ssb.no/api/klass/v1/api-guide.html
+executable: tap-ssb-klass
+keywords: []
```
Comment (pnadolny13) [inline _data/meltano/extractors/tap-ssb-klass/storebrand.yml:5]:
```suggestion
keywords:
- meltano_sdk
```

File: _data/meltano/extractors/tap-pxwebapi/storebrand.yml [hunk 3]
```diff
@@ -0,0 +1,25 @@
+capabilities: []
```
Comment (pnadolny13) [inline _data/meltano/extractors/tap-pxwebapi/storebrand.yml:1]:
```suggestion
capabilities:
- about
- batch
- catalog
- discover
- schema-flattening
- state
- stream-maps
```

File: _data/meltano/extractors/tap-ssb-klass/storebrand.yml [hunk 4]
```diff
@@ -0,0 +1,24 @@
+capabilities: []
```
Comment (pnadolny13) [inline _data/meltano/extractors/tap-ssb-klass/storebrand.yml:1]:
```suggestion
capabilities:
- about
- batch
- catalog
- discover
- schema-flattening
- state
- stream-maps
```

File: _data/meltano/extractors/tap-pxwebapi/storebrand.yml [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+logo_url: /assets/logos/extractors/pxwebapi.png
+maintenance_status: active
+name: tap-pxwebapi
+namespace: tap_pxwebapi
+next_steps: ''
+pip_url: git+https://github.com/storebrand/tap-pxwebapi.git
+quality: silver
+repo: https://github.com/storebrand/tap-pxwebapi
+settings:
+- kind: string
```
Comment (pnadolny13) [inline _data/meltano/extractors/tap-pxwebapi/storebrand.yml:17]: @hholgersen would you mind adding label and description to these settings? Those are what populate the hub page. Label is usually just a more human readable version of `name` like base_url -> Base URL

File: _data/meltano/extractors/tap-ssb-klass/storebrand.yml [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+logo_url: /assets/logos/extractors/ssb-klass.png
+maintenance_status: active
+name: tap-ssb-klass
+namespace: tap_ssb_klass
+next_steps: ''
+pip_url: git+https://github.com/storebrand/tap-ssb-klass.git
+quality: silver
+repo: https://github.com/storebrand/tap-ssb-klass
+settings:
+- kind: array
```
Comment (pnadolny13) [inline _data/meltano/extractors/tap-ssb-klass/storebrand.yml:16]: Similar to previous comment about adding label and descriptions.

**Review by pnadolny13 — COMMENTED** [review]
This is great, thanks for submitting these. I left a few comments regarding content of the plugin files if you can take a look. Also would you mind removing the diffs in the yarn lock file?

**Comment by hholgersen** [comment]
As a general matter @pnadolny13 , should we list `state` as a capability when a tap would accept the argument, but not do anything with it?

**Comment by hholgersen** [comment]
Seeing the preview, the name feels too long and clunky. Maybe just PxWebApi?

**Comment by pnadolny13** [comment]
>should we list state as a capability when a tap would accept the argument, but not do anything with it?

@hholgersen good question, I think the answer is no so it can be removed, sorry I forgot to check if you tap supported it. Although I dont think theres any negative consequences to having it, I think that capability mostly just tells meltano to track state between runs.

**Comment by hholgersen** [comment]
> > should we list state as a capability when a tap would accept the argument, but not do anything with it?
> 
> @hholgersen good question, I think the answer is no so it can be removed, sorry I forgot to check if you tap supported it. Although I dont think theres any negative consequences to having it, I think that capability mostly just tells meltano to track state between runs.

Makes sense. In that case, I think I'm good on my side. I have updated the names/labels so I'm happy with how they appear on the hub.

**Comment by pnadolny13** [comment]
@hholgersen great thanks! One last comment, is the ssb-klass logo supposed to have that black line on the left or is that from a screenshot? I noticed it in the preview https://deploy-preview-1538--meltano-hub.netlify.app/extractors/tap-ssb-klass

Screen Shot 2023-10-05 at 5 49 23 PM

**Comment by hholgersen** [comment]
> @hholgersen great thanks! One last comment, is the ssb-klass logo supposed to have that black line on the left or is that from a screenshot? I noticed it in the preview https://deploy-preview-1538--meltano-hub.netlify.app/extractors/tap-ssb-klass
> 
> Screen Shot 2023-10-05 at 5 49 23 PM

@pnadolny13 you are right of course, I didn't see that issue.  I have replaced the logos with non-screenshotted versions.

**Review by pnadolny13 — APPROVED** [review]
🚀

**Pull request merged**